bin/combine_results.py

```diff
@@ -66,20 +66,26 @@ def main():
             else:
                 #for tc in ts.getiterator("testcase"):
                 use_element.extend(list(ts));
-            
+
     if args.debug : ET.dump(result)
-    
+
+    testsuite_count = 0
+    testcase_count = 0
+
     for testsuite in result.iter('testsuite'):
+        testsuite_count += 1
         for testcase in testsuite.iter('testcase'):
+            testcase_count += 1
```

Comment:
This is better spelt using `enumerate`, as  `for testsuite_count, test_suite in enumerate(result.iter('testsuite'), 1):`